WordPress/Classes/Services/BlogSyncFacade.m

```diff
@@ -49,7 +49,7 @@ - (void)syncBlogWithUsername:(NSString *)username
     blog.password = password;
     blog.options = options;
     [[ContextManager sharedInstance] saveContext:context];
-    [blogService syncBlog:blog success:nil failure:nil];
+    [blogService syncBlog:blog];
```

Comment:
Makes sense. We'd still want to do the multi author check but another sync is triggered when the user views the blog detail so not syncing here should be fine.